internal/fourslash/tests/manual/autoImportCompletion_test.go

```diff
@@ -0,0 +1,145 @@
+package fourslash_test
```

Comment:
Technically the tests in `internal/fourslash/tests/manual/` are supposed to be generated tests that had to be manually modified, and new tests go into `internal/fourslash/tests/`, though that doesn't have practical differences I think other than our own organization of things.